ggml-metal.m

```diff
@@ -740,8 +746,12 @@ void ggml_metal_graph_compute(
                                 [encoder setBytes:&ne1  length:sizeof(ne1)  atIndex:14];
 
                                 if (src0t == GGML_TYPE_Q4_0 || src0t == GGML_TYPE_Q4_1) {
```

Comment:
should be src0t == GGML_TYPE_Q4_0 here...

Reply:
Ahh, thank you!